### ISSUE TITLE
notebook>=5.7.8 requirement in setup.py makes packages that depend on chempy unusable on Google Colab

### ISSUE THREAD

**Issue by slayoo** [bjodah/pyodesys#119]
[Google's Colab platform](https://colab.research.google.com) offers Jupyter execution in the cloud with GPU support, etc. You can do ``pip install`` there, but it is not possible to change the version of pre-installed ``notebook`` package. **pyodesys** requires notebook>=5.7.8 while Colab uses 5.3.1. 

Now, current chempy requires pyodesys>=0.14.0 which makes it unusable on Colab, and same for any package that has "chempy" within its dependencies. The workaround is to do:
```console
pip install chempy==0.7.10 pyodesys==0.13.0 notebook==5.3.1
``` 
on Colab, and that is what we will do in [PySDM-examples](https://github.com/atmos-cloud-sim-uj/PySDM-examples). Given that we get 1.5-year old chempy as a side-effect of requiring a particular version of ``notebook`` in pyodesys, it sounds as a mishap :)

Is notebook>=5.7.8 really a strong requirement here?

**Comment by bjodah** [comment]
Hi @slayoo, thank you for reporting this.

No it's not a strong requirement, I have simply just set the lower bounds based on what is (continuously) tested.

I did not know that colab uses an older version of notebook. Pull requests updating the lower bounds (and perhaps a comment mentioning Google Colab) to respective repository is most welcome. Once merged I can tag new versions and push to PyPI to hopefully ease the use on Google Colab.
